app/assets/stylesheets/v3/layout_components/responsive-table.scss

```diff
@@ -198,3 +200,80 @@ $use-respond-mixins: false !default;
     margin-bottom: 0;
   }
 }
+
+// This is the new table css as of Nov 2020. Please use this going forward.
+
+.responsive-table {
+  border-style: none;
+  border-collapse: collapse;
+  width: 100%;
+  table-layout: fixed;
+
+  thead,
+  tbody {
+    border: none;
+  }
+
+  th,
+  td {
+    padding: $base-padding;
+    text-align: left;
+    border-right: 1px solid $bg-grey-light;
+
+    &:last-child {
+      border-style: none;
+    }
+  }
+
+  th {
+    background-color: $bg-grey-lighter;
+  }
+
+  tr {
+    background-color: $white;
+  }
+
+  tbody tr:nth-child(even) {
+    background-color: $bg-grey-lightest;
+  }
+
+  @include breakpoint(small only) {
+    thead {
+      clip: rect(0 0 0 0);
+      height: 1px;
+      margin: -1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+    }
+
+    tr {
+      display: block;
+      padding: $base-padding;
```

Comment:
what do you think about having 0 padding left and right on mobile (assuming the parent container will have padding)? It starts to look a bit squished on small screens...

Reply:
I did try something similar to this, but I can take another look at it.

Reply:
So this was what it looks like removing as much of the padding as I can.
![Screen Shot 2020-11-02 at 11 25 00 AM](https://user-images.githubusercontent.com/60706495/97905684-a414d900-1cff-11eb-9046-986b2cb02790.png)

Reply:
Or we can remove the striping and get this...
![Screen Shot 2020-11-02 at 11 37 48 AM](https://user-images.githubusercontent.com/60706495/97905835-eb9b6500-1cff-11eb-86c9-c95c376fe644.png)

Reply:
ah. i forgot about the striping. i am good with leaving it how you originally had it then.